ACE/ace/config-android.h

```diff
@@ -10,14 +10,13 @@
 // There is a large number of combinations of these two that can lead to
 // problems.
 
-#if !defined (__ANDROID_API__)
-#  include <android/api-level.h>
-#endif
+#include <android/ndk-version.h>
+#include <android/api-level.h>
 
 #define ACE_ANDROID
 #define ACE_PLATFORM_CONFIG config-android.h
 
-#include "ace/config-linux-common.h"
```

Comment:
This will require the user to have the ace dir in his include path which not all ace users do

Reply:
No it doesn't. That's how `#include <>` works, but not how `#include ""` works. `#include ""` first checks to see if the path exists as file relative to the including file, then goes one to check the include path.

Reply:
I would prefer to keep `ace` in the path here for consistency, we do it everywhere else

Reply:
Alternatively we could correct it like I'm currently slowing doing in OpenDDS since it gives a false impression about how `#include` works.